src/components/Explore/utils.ts

```diff
@@ -154,13 +156,19 @@ export const EXPLORE_CHART_IDS = Object.values(exploreMetricData).map(
   metric => metric.chartId,
 );
 
+const hasValue = (point: Column) => isNumber(point.y);
+
+function cleanSeries(data: Column[]) {
+  return dropRightWhile(data, point => !hasValue(point));
```

Comment:
Hrm...  I'm not sure if this is the right fix.  This problem can happen in the front or even in the middle of a chart too.

(Nevada)
![image](https://user-images.githubusercontent.com/206364/91509449-1ba53280-e88f-11ea-8aac-1615ba816c70.png)

I think we really need to just fix it so it doesn't treat `null` as `0`

Reply:
That fix would be more difficult, as we will need to split the series into segments of contiguous non-null values, and then create independent series for them.

Maybe we can do that post-MVP?